ipaserver/plugins/baseruserfas.py

```diff
@@ -86,6 +86,19 @@
         maxlength=255,
         normalizer=lambda value: value.strip(),
     ),
+    StrEnum(
+        "faspronoun?",
+        cli_name="faspronoun",
```

Comment:
Shouldn't faspronoun also be added to default_attributes earlier in the file ?

Reply:
thanks, fixed!